src/main/java/step2/domain/Lotto.java

```diff
@@ -0,0 +1,58 @@
+package step2.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class Lotto {
+    List<Integer> lottoNumber;
+
+    public Lotto(List<Integer> lottoNumber) {
+        this.lottoNumber = lottoNumber;
+    }
+
+    public Lotto() {
+        this.lottoNumber = initLottoNumber();
+    }
+
+    public List<Integer> getLottoNumber() {
+        return lottoNumber;
+    }
+
+    private List<Integer> initLottoNumber() {
+        List<Integer> list = new ArrayList<>();
+        for (Integer i = 1; i < 46; i++) {
```

Comment:
for 문 내부에서 사용되는 인덱스는 원시타입 int를 사용하는 것이 좋습니다.
참조타입보다 성능적으로 우수해요!
```suggestion
        for (int i = 1; i < 46; i++) {
```

Reply:
1000(가격), 1, 45, 6과 같은 숫자는 Lotto 도메인에서 관리하는 상수로 만드는 것이 적절하지 않을까요?

`로또의 가격을 바꿔주세요!`, `로또 범위를 1~50까지 늘려주세요` 라는 변경 요구사항이 발생했을때, 수정해야 하는 개발자는 가장먼저 Lotto도메인을 떠올리지 않을까요? 🙂